dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/GetBucketTask.java

```diff
@@ -100,7 +100,7 @@ public GetBucketTask(final ObservableList<TreeItem<Ds3TreeTableValue>> observabl
                     .getObjects()
                     .stream()
                     .filter(c -> c.getKey() != null)
-                    .filter(c -> c.getKey().equals(ds3Value.getFullName()))
+                    .filter(c -> !c.getKey().equals(ds3Value.getFullName()))
```

Comment:
I'm a little concerned when logic like this gets inverted.  What issue was this causing?

Reply:
Losing the ! was my bad. I translated the logic to filters poorly.